pkg/grpc/default_api/api.go

```diff
@@ -57,7 +57,7 @@ var (
 		},
 	}
 	// FIXME I put it here for compatibility.Don't write singleton like this !
-	// It should be refactored and deleted.
+	// LayottoAPISingleton It should be refactored and deleted.
```

Comment:
This sentence has two subjects.
I think it should be "LayottoAPISingleton should be refactored and deleted."

Reply:
Done